### PR TITLE
EL-1219: dual deploy to new Production namespace

### PR DESCRIPTION
**Pull request by MazOneTwoOne**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1219)

## What changed and why

Dual deploy to new Production namespace
Add kube-secrets to new namespace
Add CIrcleCI context and env_vars

## Guidance to review

- Followed the detailed notes on the Jira ticket. Thanks all. 

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  containerId: GTM-WJ98FR4
+
+cfe:
+  host: http://cfe-civil.cfe-civil-staging.svc.cluster.local
+  environment_name: production
+
+featureFlags:
+  sentry: ENABLED
+  overrides: NOT_ENABLED
+  indexProduction: NOT_ENABLED
```
Comment (patrick-laa) [inline helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml:60]: Can we add `# TODO:` comments to indexProduction and basicAuthentication to remind ourselves to switch them on/off respectively when we go live with this?

File: helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  maintenanceMode: NOT_ENABLED
+  basicAuthentication: ENABLED
+  welshCw: NOT_ENABLED
+
+geckoboard:
+  metricsDataset: "ccq_monthly_metrics_staging"
```
Comment (patrick-laa) [inline helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml:66]: I hear geckoboard might be on the way out, but we should have this section pointing to the production datasets so if we switch geckoboard reporting on we push data to the right place.

File: helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  lastPagesDataset: "ccq_last_pages_staging"
+  validationsDataset: "ccq_validations_staging"
+  allJourneysDataset: "ccq_all_journeys_staging"
+  monthlyJourneysDataset: "ccq_monthly_journeys_staging"
+  recentJourneysDataset: "ccq_recent_journeys_staging"
+  enabled: not_enabled
+
+sentry:
+  # Despite looking very secret-y, the key here is _not_ a secret, as it will appear in every HTTP response header
+  cspReportEndpoint: https://o345774.ingest.sentry.io/api/6763263/security/?sentry_key=fda2f0cddce3417c87e4df62f3611e76
```
Comment (patrick-laa) [inline helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml:77]: These are staging values. I think they should be production ones.

**Review by patrick-laa — APPROVED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -355,6 +355,24 @@ workflows:
             branches:
               only:
                 - main
+      - deploy:
+          name: Deploy production to new namesapce
```
Comment (HettieS) [inline .circleci/config.yml:359]: small typo here!

Comment (MazOneTwoOne) [reply .circleci/config.yml:359]: I've also spotted this on staging. Will update both.

**Pull request merged**